src/index.html

```diff
@@ -20,8 +24,364 @@
+          <br />
+          5K Monoblock (MNED2UA/A)
+        </div>
+        <div class="card__product-code">Product code: 195434</div>
+        <div class="card__rating-field">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <div class="card__rating-text">Reviews: 5</div>
+        </div>
+        <div class="card__price-field">
+          <div class="card__price-field-text">Price:</div>
+          <div class="card__price-field-number">$2,199</div>
+        </div>
+        <a
+          class="card__buy-link"
+          href="#buy"
+        >
+          BUY
+        </a>
+      </div>
+      <div class="catalog__card card">
+        <img
+          class="card__image"
+          src="images/imac.jpeg"
+          alt="logo"
+        />
+        <div class="card__title">
+          APPLE A1419 iMac 27" Retina
+          <br />
+          5K Monoblock (MNED2UA/A)
+        </div>
+        <div class="card__product-code">Product code: 195434</div>
+        <div class="card__rating-field">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <div class="card__rating-text">Reviews: 5</div>
+        </div>
+        <div class="card__price-field">
+          <div class="card__price-field-text">Price:</div>
+          <div class="card__price-field-number">$2,199</div>
+        </div>
+        <a
+          class="card__buy-link"
+          href="#buy"
+        >
+          BUY
+        </a>
+      </div>
+      <div class="catalog__card card">
+        <img
+          class="card__image"
+          src="images/imac.jpeg"
+          alt="logo"
+        />
+        <div class="card__title">
+          APPLE A1419 iMac 27" Retina
+          <br />
+          5K Monoblock (MNED2UA/A)
+        </div>
+        <div class="card__product-code">Product code: 195434</div>
+        <div class="card__rating-field">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <div class="card__rating-text">Reviews: 5</div>
+        </div>
+        <div class="card__price-field">
+          <div class="card__price-field-text">Price:</div>
+          <div class="card__price-field-number">$2,199</div>
+        </div>
+        <a
+          class="card__buy-link"
+          href="#buy"
+        >
+          BUY
+        </a>
+      </div>
```

Comment:
The product cards from line 154 to 384 are identical in content. Consider using a template or a loop in your JavaScript to dynamically generate these cards to avoid repetition and make future updates easier.